src/middlewares/nonceMiddleware.ts

```diff
@@ -9,13 +9,13 @@ const contentSecurityPolicy = (nonce: string) => {
   const csp = `
     default-src 'self';
     connect-src 'self' https://variant.innocraft.cloud/ https://g.nav.no/api/v1/;
-    script-src 'self' 'unsafe-inline' 'nonce-${nonce}' 'strict-dynamic' https://variant.innocraft.cloud/ https://vercel.live/ ${
+    script-src 'self' 'unsafe-inline' 'nonce-${nonce}' 'strict-dynamic' ${
       process.env.NODE_ENV !== "production" ? "'unsafe-eval'" : ""
     };
     style-src 'self' 'unsafe-inline';
     img-src 'self' data: https://cdn.sanity.io/;
     media-src 'self';
-    frame-src 'self';
+    frame-src 'self' https://vercel.live/;
```

Comment:
I guess vercel.live only ever needed in preview branches? 🤔 That being said it doesn't really do a lot of harm in production build 🤷‍♂️

Reply:
Get the error in prod. Is this a sign of something else being wrong?